securedrop/sass/journalist.sass

```diff
@@ -55,6 +55,58 @@ p.breadcrumbs
   &:dir(rtl)
     text-align: right
 
+.modal-dialog
```

Comment:
What about moving this to `modules/_modal.sass` or a similar name which would then be included in `_base.sass` ? Do you envision problems if doing that ?

Reply:
Will check on this.